.travis.yml

```
@@ -14,7 +14,25 @@ before_install:
   - sudo add-apt-repository "deb http://archive.ubuntu.com/ubuntu cosmic universe"
   - sudo apt update
   - sudo apt install libopenbabel-dev
+
+  - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+  - bash miniconda.sh -b -p $HOME/miniconda
+  - export PATH="$HOME/miniconda/bin:$PATH"
+  - hash -r
+  - conda config --set always_yes yes --set changeps1 no
+  - conda update -q conda
+  - conda info -a
+  - if [[ "$PYTHON_VER" == "3.6" ]]; then
+        conda create -q -n p4env python=$PYTHON_VER ci-psi4 psi4 numpy=1.13 matplotlib jupyter scipy -c psi4;
+    else
+        conda create -q -n p4env python=$PYTHON_VER ci-psi4 psi4 numpy matplotlib jupyter scipy -c psi4/label/dev -c defaults -c conda-forge;
```

```suggestion
        conda create -q -n p4env python=$TRAVIS_PYTHON_VERSION ci-psi4 psi4 numpy matplotlib jupyter scipy -c psi4/label/dev -c defaults -c conda-forge;
```
should be the environment variable.

---

Done.